src/components/pages/courses/course-page-layout.tsx

```diff
@@ -0,0 +1,220 @@
+import * as React from 'react'
+import Link from 'next/link'
+import Image from 'next/image'
+import Markdown from 'react-markdown'
+import UserRating from 'components/pages/courses/user-rating'
+import InstructorProfile from 'components/pages/courses/instructor-profile'
+import PlayIcon from 'components/pages/courses/play-icon'
+import getDependencies from 'data/courseDependencies'
+import {get, first} from 'lodash'
+import {NextSeo} from 'next-seo'
+import removeMarkdown from 'remove-markdown'
+
+type CoursePageLayoutProps = {
+  lessons: any
+  course: any
+  ogImageUrl: string
+}
+
+const CoursePageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
+  lessons,
```

Comment:
lessons get passed in because with playlists we are loading `items` and not `lessons` specifically.